Use yuv420p10le for libsvtav1

HaveAGitGat opened the pull request.

HaveAGitGat committed:
Use yuv420p10le for libsvtav1

HaveAGitGat committed:
Apply auto-build changes

Maxmystere commented:
After doing some tests I discovered that this change is not enough, you also need to change the profile:v to main or another supported profile since main10 doesn't exist
[Supported profiles](https://www.ffmpeg.org/ffmpeg-codecs.html#Options-29)
[Detailed profiles](https://en.wikipedia.org/wiki/AV1#Profiles)

Kitt3120 commented:
For anyone waiting, I implemented this workaround for the time until this is fixed:

![image](https://github.com/user-attachments/assets/17744952-bf06-4c89-9b11-644d63188b70)
![image](https://github.com/user-attachments/assets/6102e229-d6e7-46c2-a1f4-21daf7ee1d66)

HaveAGitGat committed:
Merge branch 'master' into 10bit

HaveAGitGat committed:
fix: skip main10 profile for libsvtav1 in 10BitVideo plugin

SVT-AV1 has no main10 profile concept; ffmpeg parses -profile:v main10
as AV1 Profile 2 (4:2:2), which rejects 4:2:0 10-bit input with
'Profile 2 bit-depth < 10 requires 4:2:2 color format'. 10-bit is
conveyed via pix_fmt alone for libsvtav1.
